Common/Core/Resources/ResourcesLoader.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Text;
+
+namespace Wokhan.WindowsFirewallNotifier.Common.Core.Resources
+{
+    public static partial class ResourcesLoader
+    {
+        public static string GetMSResourceString(string src)
+        {
+            if (src?.StartsWith("@") ?? false)
+            {
+                var sb = new StringBuilder(1024); //FIXME: Hardcoded maximum string size!
```

Comment:
Think that FIXME could be removed - I assume that is just initially allocated size though.

Reply:
It allows replacing built-in MS strings (for predefined process names or alike in their rules) by the corresponding localized string.
Regarding the size, it's used right after by P/Invoke (native method) and must be kept here (as P/Invoke excepts precise buffer sizes).

Reply:
Thanks for the explanation! But how do you know that a buffer of 1024 is enough?

Reply:
It must be explained in the original win32 API documentation (or I wouldn't have used it). I agree it lacks a reference here :-)